dissect/target/plugins/os/windows/datetime.py

```diff
@@ -210,8 +226,9 @@ def check_compatible(self):
     @internal
     def tz(self, name: str) -> tzinfo:
         """Return a datetime.tzinfo of the given timezone name."""
-        tz_data_key = "HKLM\\Software\\Microsoft\\Windows NT\\CurrentVersion\\Time Zones"
-        tz_data = self.target.registry.key("\\".join([tz_data_key, name]))
+        tz_data = self.target.registry.key(
+            "\\".join(["HKLM\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Time Zones", name])
```

Comment:
```suggestion
            f"HKLM\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Time Zones\\{name}"
```
Since it's just one value?